src/Components/Form/Form.js

```diff
@@ -6,26 +6,26 @@ import foodieLogo from '../.././images/foodie-brain-logo.png'
 const POST_REVIEW = gql`
   mutation CreateReview(
     $name: String!
-    $photo: String!
+    $photo: Upload
```

Comment:
Cool to see that this ended up differently than we thought, no `string` but `upload` instead. Good work :)